user-docs/C/installation.xml

```diff
@@ -56,4 +56,105 @@
+				<listitem><para>macOS T.2 Security</para></listitem>
+			</itemizedlist>
+		</section>
+
+		<section id="starting-the-bootable-medium">
+			<title>Starting the Bootable Medium</title>
+			<para>[image placeholder]</para>
+			<para>Insert the bootable medium into the respective device (the USB drive in a USB port, the SD card in the SD card reader or the DVD disk in the DVD drive) and restart your computer. If your computer is set to first boot from the bootable medium of your choice, then the medium will automatically boot. If it doesn’t start automatically, please refer to the computer manufacturer’s documentation for entering the boot menu. This could consist of pressing a key, such as <keycap>F12</keycap>, or using a dedicated button on the computer. In some cases, you may need to boot into Windows to access the BIOS. Once you have entered the boot menu, select the appropriate boot medium device.</para>
+			<note><para>The key to press for your computer’s boot menu will depend on the manufacturer of the computer or motherboard. <keycap>F12</keycap> is the most common boot menu key, but other common alternative keys are <keycap>Escape</keycap> and <keycap>F10</keycap>. Less common keys include <keycap>F11</keycap>, <keycap>F9</keycap>, and <keycap>F8</keycap>. If your computer is not capable of booting through USB, but has a CD or DVD drive, you can utilize <ulink url="&booting-plop;">PLoP</ulink>.</para></note>
+		</section>
+
+		<section id="booting-the-installer">
+			<title>Booting the Installer</title>
+			<para>Once the bootable media starts, a blank screen will appear with a keyboard and accessibility icons [image placeholder]. At this point, you have 5 seconds to press any key to access the installer boot menu, if you wish to. The boot menu will initially present a language selection menu, followed by a simple menu with options to install or try Xubuntu, test memory, boot from the hard disk, as well as other options accessible by pressing the <keycap>F1</keycap> through <keycap>F6</keycap> keys (for more information about the Fn keys options, <ulink url="&booting-fn-keys;">click here</ulink>).</para>
+			<para>[images placeholder]</para>
+		</section>
+
+		<section id="integrity-check">
+			<title>Integrity Check</title>
+			<para>After the initial boot screen, releases of Xubuntu 20.04 and later default to checking the integrity of the install media, which can be skipped by pressing <keycap>Ctrl + C</keycap>. This check is important as it is common for corrupted installation media to cause failed installations.</para>
+			<note><para>If you get a black screen after the integrity check, reboot your system and select one of the <guilabel>safe graphics</guilabel> options in the GRUB menu.</para></note>
+			<para>[images placeholder]</para>
+		</section>
+
+	</section>
+
+	<section id="installer">
+		<title>Installer</title>
+
+		<section id="Welcome Screen">
```

Comment:
The section id's need to be in a lower-case, dash-format. In this case, "welcome-screen".

Reply:
Thanks. I forgot to fix that. New patch on the way.